src/java/org/apache/cassandra/repair/RepairRunnable.java

```diff
@@ -227,8 +227,14 @@ private void complete(String msg)
 
     public void run()
     {
+        String previousName = Thread.currentThread().getName();
         try
         {
+            Thread.currentThread().setName(NamedThreadFactory.globalPrefix()
+                                           + "Repair Coordinator #" + cmd
+                                           + " id=" + parentSession
+                                           + " ks=" + keyspace
+                                           + " tables=" + options.getColumnFamilies());
```

Comment:
Why do you say that? This is a common practice to make thread dumps easier to reason about.

Reply:
Thread dumps are only one of the tools that use thread names. Logs, JConsole, Profilers also use them. If the name of a thread keep on changing, it is pretty hard to understand what is truly going on and you cannot monitor what happen to a given thread over time.

Reply:
I disagree that it makes it harder since it provides a lot more context that enhance logs/jconsole.  most profilers won't be tricked (at least the ones I use), so reports just have more context in the names; this change is not required for the patch, so I will revert both.  

I do plan to add these back though when I start working on the repair observability tickets.